src/components/orders/GasFeeDisplay/index.tsx

```diff
@@ -45,10 +45,14 @@ export function GasFeeDisplay(props: Props): JSX.Element | null {
         {formattedExecutedFee} {quoteSymbol}
       </span>
       {/* <UsdAmount>(~${totalFeeUSD})</UsdAmount> */}
-      <span>
-        of {formattedTotalFee} {quoteSymbol}
-      </span>
-      {/* <UsdAmount>(~${executedFeeUSD})</UsdAmount> */}
+      {!fullyFilled && (
+        <>
+          <span>
+            of {formattedTotalFee} {quoteSymbol}
+          </span>
+          {/* <UsdAmount>(~${executedFeeUSD})</UsdAmount> */}
```

Comment:
Unrelated, this will be nice to add. One user submitted a code for estimating this using https://www.coingecko.com/en/api. It's ratelimiteed, so maybe is not super reliable, but it will probably be a good idea to use it for prototyping or as a fallback